doonut-app-external-api/src/main/kotlin/com/doonutmate/calendar/service/CalendarAppService.kt

```diff
@@ -0,0 +1,40 @@
+package com.doonutmate.calendar.service
+
+import com.doonutmate.calendar.controller.dto.CalendarResponse
+import com.doonutmate.calendar.controller.dto.CalendarResult
+import com.doonutmate.doonut.calendar.service.CalendarBusinessService
+import org.springframework.data.domain.PageRequest
+import org.springframework.data.domain.Pageable
+import org.springframework.data.domain.Slice
+import org.springframework.stereotype.Service
+import java.time.Instant
+
+@Service
+class CalendarAppService(
+    private val calendarBusinessService: CalendarBusinessService,
+    private val calendarFacadeService: CalendarFacadeService,
+) {
+    fun get(time: Instant?, size: Int?): CalendarResult<CalendarResponse> {
+        val pageable: Pageable = branchPageSize(size)
+
+        val calendars: Slice<CalendarResponse> = getBoards(time, pageable)
+        return CalendarResult(calendars)
+    }
+
+    private fun getBoards(time: Instant?, page: Pageable): Slice<CalendarResponse> {
+        return if (time != null) {
+            calendarFacadeService.convertToList(calendarBusinessService.findLatestCalendar(page, time))
+        } else {
+            calendarFacadeService.convertToList(calendarBusinessService.findInitialLatestCalendar(page))
+        }
+    }
+
+    private fun branchPageSize(size: Int?): Pageable {
```

Comment:
메서드는 동사로 시작하는게 좋을 것 같아요.  

```suggestion
    private fun getBranchPageSize(size: Int?): Pageable {
```